t/24-worker-overall.t

```diff
@@ -142,6 +142,26 @@ subtest 'capabilities' => sub {
     # clear cached capabilities
     delete $worker->{_caps};
 
+    subtest 'worker instance contains PARALLEL_ONE_HOST_ONLY' => sub {
+	my $worker2 = OpenQA::Worker->new({instance => 2, apikey => 'foo', apisecret => 'bar', verbose => 1, 'no-cleanup' => 1});
+        my $capabilities2 = $worker2->capabilities;
```

Comment:
You probably need to set the `PARALLEL_ONE_HOST_ONLY` setting in `$global_settings` before invoking this function.

Reply:
But if I am not wrong it should read it from https://github.com/os-autoinst/openQA/pull/5695/files#diff-90a20a69fe7c107578278a313173bef8c87bade502ecec03004c6c02eadebeeaR13

Reply:
No, it won't. Only if the instance matches and you edit the settings file that is actually relevant for this test. So far your edits don't change a relevant combination.

Note that instead of changing those ini files you can also just modify `$global_settings`. At least I think so judging by how the subsequent subtest does it.

Reply:
worked. I had a mistake in the code